### ISSUE TITLE
failed build after updating to next 13.5.1

### ISSUE THREAD

**Issue by loudmu** [prismicio/prismic-next#83]
<!-- 💙 Thanks for your time to make this package better with your feedback 💙

**IMPORTANT** Before reporting a bug please make sure that you have read through the documentation:
- https://prismic.io/docs

### Versions

- `@prismicio/next`:   "^1.3.4",
- `next`: 13.5.1
- node: 18.8

when I upgraded to next 13.5.1 a got the following fail build message error .
when i remove the  preview and exit preview everything works fine

  Creating an optimized production build .   Linting and checking validity of types 
   Linting and checking validity of types ..Failed to compile.

.next/types/app/api/exit-preview/route.ts:58:7
Type error: Type '{ __tag__: "GET"; __return_type__: Promise<void | Response>; }' does not satisfy the constraint '{ __tag__: "GET"; __return_type__: Response | Promise<Response>; }'.
  Types of property '__return_type__' are incompatible.
    Type 'Promise<void | Response>' is not assignable to type 'Response | Promise<Response>'.
      Type 'Promise<void | Response>' is not assignable to type 'Promise<Response>'. 
        Type 'void | Response' is not assignable to type 'Response'.
          Type 'void' is not assignable to type 'Response'.

  56 |         __return_type__: Response | Promise<Response>
  57 |       },
> 58 |       {
     |       ^
  59 |         __tag__: 'GET',
  60 |         __return_type__: ReturnType<MaybeField<TEntry, 'GET'>>
  61 |       },

**Comment by angeloashmore** [comment]
Hi @loudmu, could you share details about the failed build? What error are you seeing?

**Comment by angeloashmore** [comment]
Duplicate of #84.

**Comment by loudmu** [comment]
I updated to   @prismicio/next": "^1.3.5" and I still have  the same failed build problem.  when remove  Preview/route.ts and  exit-preview./route the build works . 

[package.txt](https://github.com/prismicio/prismic-next/files/12705019/package.txt)
[prismic build.txt](https://github.com/prismicio/prismic-next/files/12705020/prismic.build.txt)

**Comment by jere-co** [comment]
> I updated to @prismicio/next": "^1.3.5" and I still have the same failed build problem. when remove Preview/route.ts and exit-preview./route the build works .
> 
> [package.txt](https://github.com/prismicio/prismic-next/files/12705019/package.txt) [prismic build.txt](https://github.com/prismicio/prismic-next/files/12705020/prismic.build.txt)

Make sure to return:

```TS
-  await redirectToPreviewURL({ client, request });
+  return await redirectToPreviewURL({ client, request });
```

**Comment by loudmu** [comment]
the problem occurs only in windows10 environment . I moved my dev environment to Ubuntu and everything works perfectly.

**Comment by angeloashmore** [comment]
@loudmu Could you confirm that you have the latest `@prismicio/next` by deleting your `node_modules` folder and re-installing?

**Comment by loudmu** [comment]
problem solved after updating to next 13.5.3!
